vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/multiselect_string.tsx

```diff
@@ -40,6 +40,7 @@ const MultiselectStringInput: InputBackendComponent = ({input, onChange, onClear
       }}
       onAll={onAll}
       onClear={onClear}
+      maxItems={25}
```

Comment:
Future question now (and that's fine!) but I wonder what happens if there are 26 items?  That could happen easily if a certain project has reason to directly expose the biospecimen or subject id levels here.